### PR TITLE
Vega20 tests

### PR DESCRIPTION
**Pull request by neon60**
Vega20 tests

### CODE REVIEW

File: test/rocprim/test_utils_types.hpp [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2017 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (saadrahim) [inline test/rocprim/test_utils_types.hpp]: Copyright 2019?

Comment (neon60) [reply test/rocprim/test_utils_types.hpp]: Shouldn't it be 2017-2019?

Comment (saadrahim) [reply test/rocprim/test_utils_types.hpp]: That seems to be a new file, so 2019 might work. Please merge after any correction that you find acceptable.

Comment (neon60) [reply test/rocprim/test_utils_types.hpp]: Fixed

**Comment by saadrahim** [comment]
last question, do we need to do anything to enable these datatypes in hipCUB/rocThrust?

**Comment by ex-rzr** [comment]
> last question, do we need to do anything to enable these datatypes in hipCUB/rocThrust?

No, these types were supported from the beginning, we just added more tests just to be sure that rocPRIM+compilers work correctly with them.

**Pull request merged**